src/WinRT.Runtime/ActivationFactory.cs

```diff
@@ -190,6 +191,11 @@ public static unsafe (ObjectReference<I> obj, int hr) GetActivationFactory<I>(st
         }
     }
 
+#nullable enable

```

Comment:
For same reason as other PR, we probably should add nullable restore at the bottom the file.

Reply:
Done!